bundles/core/org.eclipse.smarthome.core.thing.test/src/test/groovy/org/eclipse/smarthome/core/thing/i18n/ThingStatusInfoI18nLocalizationServiceMultipleBundlesOSGiTest.groovy

```diff
@@ -0,0 +1,125 @@
+
+        def thingHandlerFactory = new ExtendingOtherBundleThingHandlerFactory()
+        thingHandlerFactory.activate([getBundleContext: { bundleContext }] as ComponentContext)
+        registerService(thingHandlerFactory, ThingHandlerFactory.class.name)
+
+        thing = ThingBuilder.create(new ThingTypeUID("aaa:bbb"), "ccc").build()
+
+        managedThingProvider = getService(ManagedThingProvider)
+        assertThat managedThingProvider, is(notNullValue())
+
+        managedThingProvider.add(thing)
+
+        waitForAssert({
+            assertThat thing.getStatus(), is(ThingStatus.ONLINE)
+        })
+
+        thingStatusInfoI18nLocalizationService = getService(ThingStatusInfoI18nLocalizationService)
+        assertThat thingStatusInfoI18nLocalizationService, is(notNullValue())
+    }
+
+    @After
+    void tearDown() {
+        managedThingProvider.remove(thing.getUID())
+        setDefaultLocale(defaultLocale)
+    }
+
+    @Test
+    void 'assert that translation from superclass bundle is used'() {
+        thing.setStatusInfo(new ThingStatusInfo(ThingStatus.ONLINE, ThingStatusDetail.NONE, "@text/channel-type.magic.alert.label"))
+        def info = thingStatusInfoI18nLocalizationService.getLocalizedThingStatusInfo(thing, Locale.GERMANY)
+        assertThat info, is(new ThingStatusInfo(ThingStatus.ONLINE, ThingStatusDetail.NONE, "Alarm"))
+    }
+
+    @Test
+    void 'assert that no translation is used if not found in superclass'() {
+        thing.setStatusInfo(new ThingStatusInfo(ThingStatus.ONLINE, ThingStatusDetail.NONE, "@text/some-fake-label-for-this-test"))
+        def info = thingStatusInfoI18nLocalizationService.getLocalizedThingStatusInfo(thing, Locale.GERMANY)
+        assertThat info, is(new ThingStatusInfo(ThingStatus.ONLINE, ThingStatusDetail.NONE, "@text/some-fake-label-for-this-test"))
+    }
+
+    @Test
+    void  'assert that arguments for description translation are also translated'() {
+        thing.setStatusInfo(new ThingStatusInfo(ThingStatus.ONLINE, ThingStatusDetail.NONE, "@text/offline.with-two-params [@text/channel-type.magic.alert.label, 60]"))
+        def info = thingStatusInfoI18nLocalizationService.getLocalizedThingStatusInfo(thing, Locale.GERMANY)
+        assertThat info, is(new ThingStatusInfo(ThingStatus.ONLINE, ThingStatusDetail.NONE, "Thing ist wegen Alarm offline. Bitte versuchen Sie es in 60 Sekunden erneut."))
+    }
+
+    class ExtendingOtherBundleThingHandlerFactory extends BaseThingHandlerFactory {
+
+        @Override
+        public boolean supportsThingType(ThingTypeUID thingTypeUID) {
+            return true
+        }
+
+        @Override
+        protected ThingHandler createHandler(Thing thing) {
+            new ExtendingOtherBundleThingHandler(thing)
+        }
+    }
+
+    class ExtendingOtherBundleThingHandler extends MagicColorLightHandler {
+        public ExtendingOtherBundleThingHandler(Thing thing) {
+            super(thing)
+        }
+    }
+}
```

Comment:
Please add a new line at the end of the file.

Reply:
OK, will be included in my next commit to this PR.